### ISSUE TITLE
HELM_S3_REGION does not work

### ISSUE THREAD

**Issue by yujunz** [hypnoglow/helm-s3#128]
```
argocd@argocd-repo-server-778ff5475-nzd8t:~$ helm2 s3 init s3://charts-code-nvda-ai/helm-s3
upload index to s3: upload index to S3 bucket: MissingRegion: could not find region configuration
Error: plugin "s3" exited with error
argocd@argocd-repo-server-778ff5475-nzd8t:~$ echo $HELM_S3_REGION
us-west-1
```

**Comment by JihedBenYedder** [comment]
I have faced the same issue and i saw that this is the PR to fix it but it is not merged yet and it is an old one... can you merge it please ?

**Comment by JihedBenYedder** [comment]
I have faced the same issue and i saw that this is the PR to fix it but it is not merged yet and it is an old one... can you merge it please ?

**Comment by hypnoglow** [comment]
The issues with region should be resolved since v0.11.0. I'm going to close this.
